Containers/talk/Dockerfile

```diff
@@ -1,18 +1,16 @@
 FROM nats:2.9.18-scratch as nats
 FROM strukturag/nextcloud-spreed-signaling:1.1.2 as signaling
-FROM coturn/coturn:4.6.2-r3-alpine
-USER root
-# Pin alpine version manually as long as https://github.com/coturn/coturn/issues/1226 is not done
-ENV ALPINE_VERSION=3.18
+FROM ghcr.io/processone/eturnal:x.x.x-otpxx AS eturnal
```

Comment:
This needs to be released then first

Reply:
yes, probably in July or August. Is there any schedule from your side?

Reply:
> Is there any schedule from your side?

No not really. Just added this comment so that we dont forget it :D

Reply:
yeah, okay. 
Well, regarding the open items below, like whitelist, ip relay addresses, internally/externally, we can look again in the next week as we need a better understanding of how your components talk to each other. 
Is that a deal?

Reply:
Yes, thank you very much for all the help!

Reply:
Just, as I did not think about it yesterday, to be independent from Docker images we provide, you can also use our Alpine package directly (it is maintained by us as well, note the differences of the lib files):

```
FROM docker.io/library/alpine:3.18 AS build
RUN apk add eturnal --no-cache

RUN ln -s /opt/eturnal/bin/stun /usr/local/bin/stun
RUN mkdir /usr/lib/eturnal/etc /run/eturnal
RUN mv /etc/eturnal.yml /usr/lib/eturnal/etc/eturnal.yml
RUN chown -R eturnal:eturnal /usr/lib/eturnal

USER eturnal
CMD ["eturnalctl", "foreground"]
```

The package installs the eturnal user already, so it is present. Installing eturnal this way you do not have the STUN_SERVICE query included in eturnalctl, but that is actually also not included currently either in your build, since you use the configuration file for specifying the relay addresses.

Reply:
also an option

Reply:
Thanks for the idea! However I suppose we cannot easily pin a specific version then and use dependabot to update that?

An additional question: with coturn we currently also use NC_DOMAIN:3478 as STUN server. Is this also possible with eturnal?

Reply:
eturnal integrates STUN as well, so yes!

Reply:
`apk` can used with `apk add eturnal=1.10.1`. Not sure if dependabot understands that though. :)

Reply:
> eturnal integrates STUN as well, so yes!

Great, Thanks! :)

Reply:
Regarding packages, I would like to wait for the docker container :)

Reply:
Hello,

Hope you are doing alright.
```
FROM ghcr.io/processone/eturnal:x.x.x-otpxx AS eturnal
```

should be changed to 
```
FROM ghcr.io/processone/eturnal:x.x.x AS eturnal
```

We have merged the Fork today into the repository. The next release is soon with the tag `1.10.2`

Reply:
We plan for the beginning of next week.

Reply:
fyi: `1.11.1` is released. The image name is `ghcr.io/processone/eturnal:1.11.1`.

Reply:
many thanks!

Reply:
rebased, many thanks!